tests/aggregation/tests.py

```diff
@@ -1258,11 +1258,10 @@ def test_annotate_over_annotate(self):
         self.assertEqual(author.sum_age, other_author.sum_age)
 
     def test_aggregate_over_aggregate(self):
-        msg = "Cannot compute Avg('age'): 'age' is an aggregate"
-        with self.assertRaisesMessage(FieldError, msg):
-            Author.objects.annotate(age_alias=F("age"),).aggregate(
-                age=Sum(F("age")),
-                avg_age=Avg(F("age")),
+        with self.assertRaises(FieldError):
```

Comment:
The error message changed here, mentioning that `age_agg` refers to a missing annotation or field. I guess `get_aggregation` could catch the `FieldError` and re-raise with the original message if the missing field is in `aggregates` if we wanted to.